src/rules/selector-no-redundant-nesting-selector/README.md

```diff
@@ -59,3 +59,51 @@ p {
   }
 }
 ```
+
+## Options
```

Comment:
This should follow the formatting for options that is used in `stylelint` and other `stylelint-scss` rules, e.g. https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/at-function-named-arguments/README.md#options